localized/games/eggquiz/eggquizliteracy_levels_en.tsv

```diff
@@ -637,10 +637,10 @@ PostTest_9	1	9001, 9002, 9003, 9004, 9005, 9006, 9007, 9008, 9009, 9010
+			9007	imageseq_sentence		I am mad. I had a bad day at school.		Your day wasn't all bad.	I am mad. I had a bad day at school. Miss Divya helped me up. Your day wasn't all bad.	“Miss Divya helped me up.”	“I tripped over it.”^“Miss Divya helped me up.”^“This boy laughed at me.”
+			9008	imageseq_sentence		book_en_413_page_4.png		book_en_413_page_8.png	We can bring melons to eat on the moon. We will eat them with spoon. Ajay said to Richa. Ajay and Richa built rockets boots with some tools. They ate their breakfast of melon with the spoon.	Ajay and Richa build rockets boots with some tools.	Ajay and Richa build rockets boots with some tools.^Ajay and Richa knew they must be home soon.^They landed back at home around noon.
 			9009	imageseq_image		book_en_507_page_1.png		book_en_507_page_10.png	Once there was a man named Parvez. who was much too tall for his own good. Life was hard for Parvez and the villagers felt bad for him. Now. Parvez is very happy. He lives his life easily and it is all thanks to his friends in the village.	book_en_507_page_5.png	book_en_507_page_8.png^book_en_507_page_5.png^book_en_507_page_9.png
-			9010	paragraph_sentence	Why did Lion summon Elephant to his log?	In the evening, Elephant was thirsty. She went back to the watering hole. When she tried to get a drink, several frogs got in her way. She accidentally stepped on two of them. Lion summoned Elephant to his log. You are very large, he said. You must be careful of the frog because they are very small.			In the evening. Elephant was thirsty. She went back to the watering hole. When she tried to get a drink. several frogs got in her way. She accidentally stepped on two of them. Lion summoned Elephant to his log. You are very large. he said. You must be careful of the frogs. because they are very small.	To tell him to be careful.	To tell him to get a drink.^To tell him to leave the savnitika.^To tell him to be careful.
+			9010	paragraph_sentence	Why did Lion summon Elephant to his log?	In the evening, Elephant was thirsty. She went back to the watering hole. When she tried to get a drink, several frogs got in her way. She accidentally stepped on two of them. Lion summoned Elephant to his log. You are very large, he said. You must be careful of the frog because they are very small.			In the evening. Elephant was thirsty. She went back to the watering hole. When she tried to get a drink. several frogs got in her way. She accidentally stepped on two of them. Lion summoned Elephant to his log. You are very large. he said. You must be careful of the frogs. because they are very small.	To tell him to be careful.	To tell him to get a drink.^To tell him to leave the savanna.^To tell him to be careful.
```

Comment:
Check if tab or space is added in this line

Reply:
No tab or space is added, tested the updated TSV in English build